client-src/elements/chromedash-gate-column.js

```diff
@@ -209,10 +209,13 @@ export class ChromedashGateColumn extends LitElement {
     const postToThreadType = (
         this.postToThreadRef.value?.checked ? this.gate.gate_type : 0);
     if (commentText != '') {
-      window.csClient.postComment(
-        this.feature.id, this.gate.id, commentText,
-        Number(postToThreadType))
-        .then(() => this.reloadComments());
+      Promise.all([
+        window.csClient.postComment(
+          this.feature.id, this.gate.id, commentText, Number(postToThreadType)),
+        window.csClient.setVote(featureId, this.gate.id, null),
```

Comment:
There is actually a separate "Save" button that appears right next to the vote select widget.  So, we don't want the "Post" button to save the vote.